test/test_unit.jl

```diff
@@ -546,7 +552,7 @@ end
         @test cons_vars ≈ entropy2cons(entropy_vars, equations)
 
         total_energy = energy_total(cons_vars, equations)
-        @test total_energy ≈ entropy(cons_vars, equations)
+        @test total_energy ≈ Trixi.entropy(cons_vars, equations)
```

Comment:
Why this change?

Reply:
Does `using Convex: Convex` or `import Convex` work as well to trigger the loading of the extension?

Reply:
`using Convex: Convex triggers` the extension just fine too. I have made adjustments according to your recommendation. Thank you!